examples/dom/WebReconciler.re

```diff
@@ -5,16 +5,20 @@
   * This is just an example but you could use this to create interesting
   * CLI apps, with a react-like functional API!
 */
+
+exception InvalidNodePrimitiveMatchInUpdateInstance;
+
+let str = string_of_int;
+
 module Reconciler = {
   /*
      Step 1: Define primitives
    */
-  type imageProps = {src: string};
-  type buttonProps = {src: string};
   type primitives =
-    | Div
-    | Span(string)
-    | Image(imageProps);
+    | View
```

Comment:
Exploring the feasibility of making the DOM reconciler primitives more similar than Revery ones 😄